src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TraineeAPI.cs

```diff
@@ -32,10 +32,7 @@ public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
 
             var created =await repository.CreateAsync(new Trainee { FirstName = trainee.FirstName, LasttName = trainee.LasttName, EmailAddress = trainee.EmailAddress });
```

Comment:
This line can be simply written as `var created = await repository.CreateAsync(trainee);`

Reply:
Done